frontend/src/context/Metrics/metricsSlice.tsx

```diff
@@ -60,6 +66,15 @@ export const metricsSlice = createSlice({
       })
     },
 
+    updateMetricsState: (state, action) => {
+      state.isProjectCreated = action.payload.isProjectCreated
```

Comment:
Store action.payload as a variable, use it below steps,reduce the performance cost

Reply:
Is that mean to deconstruct payload?